v4.5/Makefile

```diff
@@ -0,0 +1,322 @@
+PROFILE := v4.5
```

Comment:
I am not concerned by not shared code but still would like to know which version of .net 4.5 this is